webapp/packages/core-blocks/src/FormControls/Filter.tsx

```diff
@@ -106,10 +112,12 @@ export const Filter = observer<ControlledProps | ObjectsProps<any, any>>(functio
   const styles = useStyles(filterStyles, style, toggleMode && toggleModeButtonStyle);
   const [inputRef, ref] = useFocus<HTMLInputElement>({});
   const [toggled, setToggled] = useState(!toggleMode);
+  const [isFilled, setIsFilled] = useState(false);
 
   const filter = useCallback(
     (value: string | number, name?: string) => {
       value = String(value);
+      setIsFilled(!!value);
```

Comment:
You probably need to use `value` and if it's not empty or undefined then we can reset the value